### ISSUE TITLE
[1.4.6] Dont import ProductInstaller classes which don't exists

### ISSUE THREAD

**Issue by mj1985** [oveleon/contao-recommendation-bundle#25]
### Prerequisites

- [X] I [checked](https://github.com/oveleon/contao-recommendation-bundle/issues?q=is%3Aissue) for duplicate issues (open and closed)
- [X] I am using the latest stable [version/release](https://packagist.org/packages/oveleon/contao-recommendation-bundle) of the Contao Recommendation Bundle
- [X] I have read the [contributing guidelines](https://github.com/oveleon/contao-recommendation-bundle#bugs-and-feature-requests)

### Please select the topic(s) that most closely match your concern

Backend (PHP)

### Description

We don't have `oveleon/product-installer` installed in our contao installation (and it's not required in the `composer.json` either) and after updating to `1.4.6` the installation fails, because the update tries to import classes which don't exists. See log output.

### Relevant log output

```shell
Class "Oveleon\ProductInstaller\Import\Validator\ValidatorInterface" not found while loading "Oveleon\ContaoRecommendationBundle\Import\Validator\RecommendationValidator"
```

**Comment by zoglo** [comment]
@mj1985 Could you lock your version to `1.4.5` for now?

**Comment by mj1985** [comment]
Thanks for the new branch, we'll test it tomorrow.

**Comment by zoglo** [comment]
Fixed in 1.4.7 @mj1985
